packages/react-import-remote/src/test/ImportRemote.test.tsx

```diff
@@ -77,27 +97,31 @@ describe('<ImportRemote />', () => {
 
       const newSource = 'https://bar.com/foo.js';
 
-      importRemote.setProps({source: newSource});
+      // we need to have the async here otherwise we will get
+      // a console error an `act(...)` here
+      // eslint-disable-next-line require-await
+      await importRemote.act(async () => {
+        importRemote.setProps({source: newSource});
+      });
+
       expect(load).toHaveBeenCalledWith(newSource, mockProps.getImport, nonce);
     });
   });
-  // async act is available in React 16.9.0, when we have upgraded the following
-  // tests can be enabled. https://github.com/Shopify/quilt/pull/688
-  // eslint-disable-next-line jest/no-disabled-tests
-  describe.skip('onImported()', () => {
+
+  describe('onImported()', () => {
     it('is called with the result of loading the script when successful', async () => {
       const result = 'foo';
-      const promise = Promise.resolve(result);
       const spy = jest.fn();
-      load.mockImplementation(() => promise);
+      const resolve = createResolvableLoad(result);
 
-      const importRemote = await mount(
+      const importRemote = mount(
         <ImportRemote {...mockProps} onImported={spy} />,
       );
 
-      importRemote.act(async () => {
-        await promise;
+      await importRemote.act(async () => {
+        await resolve;
```

Comment:
Not called as a function here?

Reply:
Makes me think that this is kind of a placebo, and all it's doing is pushing the act to include the next event loop tick?